backend/src/main/java/org/cenoteando/CrossOriginConfig.java

```diff
@@ -0,0 +1,24 @@
+package org.cenoteando;
+
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class CrossOriginConfig {
```

Comment:
I'm not sure if we really need this file. Let me know if it's not necessary to remove it.